### DISCUSSION TITLE
Arguments?

### DISCUSSION THREAD

**Discussion by DiggyDre** [vladmandic/sdnext#280] [Q&A]
Where do I set arguments such as --xformers or a path to my models to that I don't have to copy them again?

**Comment by vladmandic** [comment]
read the docs/notes.
its pretty much all in ui settings now.

**Reply by myndxero** [reply]
Setting the directories in settings was a godsend, wasn't always clear which ones I could define in auto1111's. Since I run multiple instances, I like to have them all centralized instead a separate set for every one. I've tons of space, but not tons for days.

**Comment by DiggyDre** [comment]
Awesome! Thanks!
